src/e-pr.js

```diff
@@ -50,8 +65,9 @@ try {
 
 program
   .description('Open a GitHub URL where you can PR your changes')
+  .option('-r, --repo <repository>', 'Repository to use', defaultRepo)
```

Comment:
generally i feel one-line flags like this are intuitively assumed to take single-string arguments, so it may not be intuitive that it should be `--repo=codebytere/repo` and not just `--repo=codebytere` or something else. Perhaps we could make this more clear?